src/util/util/Email.ts

```diff
@@ -43,3 +51,360 @@ export function adjustEmail(email?: string): string | undefined {
 
 	// return email;
 }
+
+const transporters = {
```

Comment:
Maybe move these into separate files instead, maybe into smth like src/util/email/transports and email.ts can go in src/util/email? smth like that